### PR TITLE
Updating ZPA to use zobject

### PR DESCRIPTION
**Pull request by jeff-powell**
Fix for Zing-13679
https://jira.zenoss.com/browse/ZING-13679

### CODE REVIEW

**Comment by webhookbotuser** [comment]
Can one of the admins verify this patch?

**Comment by webhookbotuser** [comment]
Can one of the admins verify this patch?

**Comment by webhookbotuser** [comment]
Can one of the admins verify this patch?

File: Products/ZenPackAdapter/monkey.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from Products.ZenUtils.CmdBase import CmdBase
 from Products.ZenPackAdapter.cloudpublisher import CloudMetricPublisher
 
+import zenwrapt
```
Comment (joshw) [inline Products/ZenPackAdapter/monkey.py:26]: doesn't appear to be used

File: Products/ZenPackAdapter/services.py [hunk 2]
```diff
@@ -68,6 +67,11 @@ def inner(*args, **kw):
     return inner
 
 
+def get(device=None, component=None):
```
Comment (joshw) [inline Products/ZenPackAdapter/services.py:70]: probably doesn't need to be here.

**Review by joshw — CHANGES_REQUESTED** [review]

File: Products/ZenPackAdapter/monkey.py (hunk 1 above)
Comment (jeff-powell) [reply Products/ZenPackAdapter/monkey.py:26]: zenwrapt auto-initialized when imported, so nothing more needed to be done. Originally, that code was included in monkey.py, but was separated out for readability and maintainability.

I update zenwrapt.py to have an initialize() method that is now called here and should be more intuitive.

File: Products/ZenPackAdapter/services.py (hunk 2 above)
Comment (jeff-powell) [reply Products/ZenPackAdapter/services.py:70]: removed - good catch
I think this was left over from when I was comparing the new zobject devices to the (previous) db object devices.

**Review by joshw — APPROVED** [review]

**Pull request merged**